package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@rapideditor/location-conflation",
   "version": "2.0.0",
   "license": "ISC",
-  "repository": "github:rapideditor/location-conflation",
+  "repository": "https://github.com/rapideditor/location-conflation",
```

Comment:
despite [the npm docs allowing this syntax](https://docs.npmjs.com/cli/v11/configuring-npm/package-json#repository), the npm website doesn't allow it.... Therefore, [the link from the npm package](https://npm.im/@rapideditor/location-conflation) to the git repo no longer works